Changelog.md

```diff
@@ -27,6 +27,7 @@ Bugfixes:
  * Type System: Improve error message when attempting to shift by a fractional amount.
  * Type System: Make external library functions accessible.
  * Type System: Prevent encoding of weird types.
+ * Commandline interface: fix error messages for imported files that don't exist
```

Comment:
Please use full sentences, start with capital letters and avoid contractions :)

Reply:
Ok :-).